storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -65,11 +65,8 @@ bool Redo_Log_Reader::find_start_checkpoint_lsn() {
   /* Redo log file header is never encrypted. */
   Encryption_metadata unused_encryption_metadata;
 
-  Log_files_context log_files_ctx =
-      Log_files_context{srv_log_group_home_dir, Log_files_ruleset::CURRENT};
-
   auto file_handle =
-      Log_file::open(log_files_ctx, checkpoint.m_checkpoint_file_id,
+      Log_file::open(log_sys->m_files_ctx, checkpoint.m_checkpoint_file_id,
```

Comment:
where do we populate log_sy->m_files_ctx?

Reply:
at the end'ish of log_sys_init in log0log.cc